src/components/modal/ModalPro.vue

```diff
@@ -117,7 +140,7 @@ module.exports = {
 			if (quotaBytes >= bytes && bytes > 0) {
 			    that.updatePayment()
 			    that.$store.commit("SET_MODAL", false)
-			    that.$toast.info('Thank you for signing up to a Peergos Pro account!',{timeout:false, id: 'pro'})                            
```

Comment:
maybe add an exceptionally block on requestSpace to handle unforseen/network errors. Nothing worse then not knowing if request completed.